_layouts/person.html

```diff
@@ -1,81 +1,156 @@
-    {% endif %}
-
-    {% if page.person.links.facebook_page %}
-    <dt>Facebook</dt>
-    <dd><a rel="nofollow" href="{{ page.person.links.facebook_page.url }}">{{page.person.links.facebook_page.url}}</a> <small>(campaign page)</small></dd>
-    {% endif %}
-
-    {% if page.person.links.homepage %}
-    <dt>Homepage</dt>
-    <dd><a rel="nofollow" href="{{page.person.links.homepage.url}}">{{page.person.links.homepage.url}}</a></dd>
-    {% endif %}
-
-    {% if page.person.links.wikipedia %}
-    <dt>Wikipedia page</dt>
-    <dd><a rel="nofollow" href="{{page.person.links.wikipedia.url}}">{{page.person.links.wikipedia.url}}</a></dd>
+        They
+    {% endif %} 
+    previously ran in
+    {% if page.person.candidacies.ge2010.constituency.post_id == page.person.candidacies.ge2015.constituency.post_id %}
+        the same constituency
+    {% else %}
+        {{ page.person.candidacies.ge2010.constituency.name }}
     {% endif %}
+    in 2010,
 
-    {% if page.person.links.linkedin %}
-    <dt>LinkedIn page</dt>
-    <dd><a rel="nofollow" href="{{page.person.links.linkedin.url}}">{{page.person.links.linkedin.url}}</a></dd>
+    {% if page.person.candidacies.ge2010.party.id == page.person.candidacies.ge2015.party.id %}
+    also as a {{ page.person.candidacies.ge2010.party.name }} candidate.
+    {% else %}
+    as a {{ page.person.candidacies.ge2010.party.name }} candidate.
     {% endif %}
+  {% endif %}
+  </p>
+ 
+  {% if page.person.biography %} 
+  <div class='person__bio'>
+    <span class='person__box__byline'>jacked from <a href='{{ page.person.links.wikipedia.url }}'>Wikipedia</a> <small><a href='http://en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License'>CC BY-SA 3.0</a></small></span>
+    <h2><i class='fa fa-book'></i> Biography</h2>
+    <p>{{ page.person.biography }}</p>
+  </div>
+  {% endif %}
 
-    {% if page.person.links.party_PPC_page %}
-    <dt>Party PPC page</dt>
-    <dd><a rel="nofollow" href="{{page.person.links.party_PPC_page.url}}">{{page.person.links.party_PPC_page.url}}</a></dd>
-    {% endif %}
+  <div class='person__cv'>
+  <span class='person__box__byline'>by <a href='https://cv.democracyclub.org.uk/'>Democracy Club CVs</a></span>
+  <h2><i class='fa fa-file-text-o'></i> CV</h2>
+  {% if page.person.cv %}
+    <a class='person__cv__imglink' href="http://cv.democracyclub.org.uk/show_cv/{{ page.person.id }}">
+      <img src="{{ page.person.cv.thumb.url }}">
+    </a>
+
+    <p class='person__cv__text'>
+        {{ page.person.name}} has sent us {{ possessive|downcase }} CV. 
+
+        <a href="http://cv.democracyclub.org.uk/show_cv/{{ page.person.id }}">
+        Read {{ page.person.name}}'s CV now.
+        </a>
+    </p>
+    <div style='clear:both;'></div>
+  {% else %}
+    <p class='person__cv__text'>{{ page.person.name }} hasn't sent <a href='cv.democracyclub.org.uk'>Democracy Club CVs</a> {{ possessive|downcase }} CV.</p>
```

Comment:
`XX hasn't sent Democracy Club CVs her CV.` is a bit clunky – maybe `XXX hasn't sent us her CV`?  I don't think anyone will know why the extra words are in there.